workflows/smrnaseq.nf

```diff
@@ -110,6 +111,14 @@ workflow NFCORE_SMRNASEQ {
 
     ch_fasta = params.fasta ? file(params.fasta): []
     ch_reads_for_mirna = FASTQ_FASTQC_UMITOOLS_FASTP.out.reads
+    // Trim 3' end nucleotides after adapter is removed, otherwise they are not really trimmed
+    if (params.three_prime_clip_r1){
+        FASTP3(
+            ch_reads_for_mirna
+        )
+        ch_reads_for_mirna  = FASTP3.out.reads
+        //trim_json         = FASTP3.out.json
```

Comment:
Can this line be removed instead of commented?

Reply:
yes, I can do that